app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointmentRow.kt

```diff
@@ -131,10 +136,23 @@ data class OverdueAppointmentRow(
         "$overdueDays"
     )
 
+    holder.diagnosisTextView.text = diagnosisText()
+
     updateBottomLayoutVisibility(holder)
     updatePhoneNumberViewVisibility(holder)
   }
 
+  private fun diagnosisText(): CharSequence {
+    return listOf(
+        diagnosedWithDiabetes to "Diabetes",
+        diagnosedWithHypertension to "Hypertension"
+    )
+        .filter { (answer, _) -> answer is Answer.Yes }
+        .map { (_, diagnosisTitle) -> diagnosisTitle }
+        .ifEmpty { listOf("None") }
```

Comment:
Please check if this is the expected behavior. If so, please add this string to `strings.xml`

Reply:
I had spoken to Pragati and we had decided that we'll show "None". But now that Pragati will show us new designs this might change again tomorrow.

Reply:
But I have put it under string.xml regardless